test/basic.test.cjs

```diff
@@ -51,4 +51,9 @@ test('Should produce normalize files', async t => {
 test('Should produce optional mask props', async t => {
   t.assert(fs.existsSync('./masks.edges.min.css'))
   t.assert(fs.existsSync('./masks.corner-cuts.min.css'))
+})
+
+test('Should produce typings files', async t => {
+  t.assert(fs.existsSync('../dist/open-props.d.ts'))
+  t.assert(fs.existsSync('../dist/props.sizes.d.ts'))
```

Comment:
i tried on stackblitz and it seems like all tests using `existsSync` are failing

Reply:
@argyleink - i got the tests to pass locally after `npm run build; npm run bundle;` (it took a while, but stackblitz ran out of memory in my browser) by changing `../` to `./`: 

```suggestion
test('Should produce typings files', async t => {
  t.assert(fs.existsSync('./dist/open-props.d.ts'))
  t.assert(fs.existsSync('./dist/props.sizes.d.ts'))
```
if i understand correctly, from ava's point of view, it's searching for files from top-level folder, not from the point of view of the test file